src/algebra/group/basic.lean

```diff
@@ -285,21 +284,22 @@ by simp only [mul_assoc, mul_inv_rev, div_eq_mul_inv]
 
 end division_monoid
 
-namespace division_comm_monoid
+section division_comm_monoid
 variables [division_comm_monoid α] (a b c d : α)
 
 local attribute [simp] mul_assoc mul_comm mul_left_comm div_eq_mul_inv
 
 @[to_additive neg_add] lemma mul_inv : (a * b)⁻¹ = a⁻¹ * b⁻¹ := by simp
+@[to_additive] lemma inv_div' : (a / b)⁻¹ = a⁻¹ / b⁻¹ := by simp
 @[to_additive] lemma div_eq_inv_mul : a / b = b⁻¹ * a := by simp
 @[to_additive] lemma inv_mul_eq_div : a⁻¹ * b = b / a := by simp
 @[to_additive] lemma inv_mul' : (a * b)⁻¹ = a⁻¹ / b := by simp
-@[to_additive] lemma inv_div_inv : (a⁻¹ / b⁻¹) = b / a := by simp
+@[simp, to_additive] lemma inv_div_inv : (a⁻¹ / b⁻¹) = b / a := by simp
```

Comment:
If `simp` can solve this, then surely it doesn't need to be marked `simp`?

Reply:
Notice
```
local attribute [simp] mul_assoc mul_comm mul_left_comm div_eq_mul_inv
```
on line 290. Also, this is just is a lemma I forgot to simp. cf [the current `inv_div_inv`](https://leanprover-community.github.io/mathlib_docs/algebra/group/basic.html#inv_div_inv).